internal/storage/storage_handle_test.go

```diff
@@ -25,79 +25,68 @@ import (
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
 	"github.com/jacobsa/oglematchers"
 	. "github.com/jacobsa/ogletest"
```

Comment:
should we also use replace assertions with https://pkg.go.dev/github.com/stretchr/testify/assert#hdr-Assertions  and get remove ogletest and oglematchers?

Reply:
Sorry, I initially thought so, but then I realized that these assert statements are part of a testing package.
Fixed.